rpc_jobs/re_release_manual.yml

```diff
@@ -20,6 +20,8 @@
           SLAVE_CONTAINER_DOCKERFILE_REPO: "RE"
           SLAVE_CONTAINER_DOCKERFILE_PATH: "./Dockerfile.standard_job"
           SLAVE_CONTAINER_DOCKERFILE_BUILD_ARGS: "BASE_IMAGE=ubuntu:16.04"
+          hold_on_error: false
+          hold_on_error_duration: "nohold"
```

Comment:
If `hold_on_error` is `true` and `hold_on_error_duration` is `"nohold"`, what happens?

Does `"nohold"` have a special significance? I can't find it defined anywhere. Why not just use `0` or maybe `null`?

I'm inclined to suggest that either `hold_on_error_duration` should always have a valid time or `hold_on_error` should be ditched and `hold_on_error_duration` only be used to determine what happens.

Reply:
I agree, collapsed into a single variable.